.github/ci_setup.sh

```diff
@@ -21,5 +21,6 @@ if [ "${PLATFORM_NAME}" = "SYCL" ]; then
    export CC=`which clang`
    export CXX=`which clang++`
    export SYCLCXX="${CXX} -fsycl"
-   export SYCL_DEVICE_FILTER=host
+   export ONEAPI_DEVICE_SELECTOR=host
```

Comment:
I don't think this will work... 🤔 You'll rather need:

```suggestion
   export ONEAPI_DEVICE_SELECTOR=opencl:cpu
```

Similar to for instance: https://github.com/acts-project/traccc/blob/main/.gitlab-ci.yml#L86

See: https://intel.github.io/llvm/EnvironmentVariables.html#oneapi-device-selector

Reply:
Changed